pkgs/development/python-modules/configargparse/default.nix

```diff
@@ -25,12 +25,15 @@ buildPythonPackage rec {
     pyyaml
   ];
 
-  pythonImportsCheck = [ "configargparse" ];
+  pythonImportsCheck = [
+    "configargparse"
+  ];
 
   meta = with lib; {
     description = "A drop-in replacement for argparse";
     homepage = "https://github.com/bw2/ConfigArgParse";
+    changelog = "https://github.com/bw2/ConfigArgParse/releases/tag/v${version}";
```

Comment:
@fabaff In which sense is that a changelog? Upstream doesn't publish release notes with their tags.

Reply:
They [do](https://github.com/bw2/ConfigArgParse/releases)...unfortunately they have a different standard than others for documenting changes.

Reply:
It's only the commit message of the tagged commit. That's really not a changelog